### PR TITLE
6th Feb - Charu

### PR DESCRIPTION
**Pull request by gem-charugarg**
Ayush Mathur

### CODE REVIEW

**Review by gem-ayushmathur — APPROVED** [review]

File: src/main/resources/features/Login.feature [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Feature: Login
+
+  Scenario:User verifies Login Page Elements Before Login
+    Then Verify all the elements present on Login Page
+
+  Scenario: User verifies elements of Login page
```
Comment (gem-ayushmathur) [inline src/main/resources/features/Login.feature:6]: Repeating scenario

File: src/main/resources/features/Login.feature [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    Then Verify all the elements present on Login Page
+
+  Scenario Outline: User Clicks on Sign In without entering Credentials
+    Given Enter username "<uname>"
```
Comment (gem-ayushmathur) [inline src/main/resources/features/Login.feature:10]: @Given @When @Then steps standard are not followed.

**Review by gem-ayushmathur — CHANGES_REQUESTED** [review]
Added my comments where changes are required.

**Comment by gem-charugarg** [comment]
@gem-ayushmathur Please review this one.

**Review by gem-ayushmathur — APPROVED** [review]
Approved @gemini-akashGarg

**Pull request merged**